File: PYTHON/1115.py

while(True):
    q1,q2 = map(int, input().split())
    if q1 == 0 or q2 ==0:
        break
    if q1>0 and q2>0:
        print("primeiro")
    if q1>0 and q2<0:
        print("quarto")
    if q1<0 and q2<0:
       print("terceiro")
    if q1<0 and q2>0:
        print("segundo")

File: CPP/1115.cpp
#include<iostream>
using namespace std;

int main(){
    int q1,q2;

     while (true)
    {
        cin>>q1>>q2;

        if(q1 == 0 || q2 == 0)
            break;
        if(q1>0 && q2 >0 )
            cout<<"primeiro\n";
        if(q1<0 && q2 >0 )
            cout<<"segundo\n";
        if(q1>0 && q2 <0 )
            cout<<"quarto\n";
        if(q1<0 && q2 <0 )
            cout<<"terceiro\n";

    }
    

    return 0;
}